src/constants/externalResources.test.ts

```diff
@@ -78,3 +79,18 @@ describe("getTaskSystemMetricsUrl", () => {
     );
   });
 });
+
+describe("getTaskSystemMetricsUrlWithDisks", () => {
+  it("generates the correct url", () => {
```

Comment:
I'd recommend just adding this test to the block above, since they test the same function! And renaming the block to `getHoneycombSystemMetricsUrl` (maybe `getTaskSystemMetricsUrl` was the old name?)

Reply:
done